Eat_and_Greet/src/comps/GrowingTextbox/textbox.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+import { TextInput} from 'react-native';
+
+export default class GrowingTextbox extends Component {
+
+  constructor (props) {
+    super(props);
+    this.state = {
+      value: '',
+      height: 40
+    }
+  }
+
+  updateSize = (height : number) => {
+    this.setState({
+      height
+    });
+  }
+
+  handleText = (value: string) => {
+    this.setState({value});
+    this.props.onTextChange(value);
+  }
+
+  render () {
+    const {value, height} = this.state;
+
+    let newStyle = {
+      height : height
+    }
+
+    return (
+    <TextInput
+      placeholder="Your Placeholder"
+      onChangeText={(value) => this.handleText(value)}
+      style={newStyle}
+      editable={true}
+      multiline={true}
+      value={value}
+      onContentSizeChange={(e) => this.updateSize(e.nativeEvent.contentSize.height)}
```

Comment:
Not a review but curiousity, How does this get called? Everytime the text changes, or when it starts a new line? Either way well done

Reply:
I believe its new line, its just a parameter surprisingly enough!